perun-core/src/main/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_user_attribute_def_virt_mfaStatus.java

```diff
@@ -0,0 +1,56 @@
+package cz.metacentrum.perun.core.impl.modules.attributes;
+
+import cz.metacentrum.perun.core.api.Attribute;
+import cz.metacentrum.perun.core.api.AttributeDefinition;
+import cz.metacentrum.perun.core.api.AttributesManager;
+import cz.metacentrum.perun.core.api.User;
+import cz.metacentrum.perun.core.api.exceptions.AttributeNotExistsException;
+import cz.metacentrum.perun.core.api.exceptions.WrongAttributeAssignmentException;
+import cz.metacentrum.perun.core.impl.PerunSessionImpl;
+import cz.metacentrum.perun.core.implApi.modules.attributes.UserVirtualAttributesModuleAbstract;
+import org.slf4j.LoggerFactory;
+import org.slf4j.Logger;
+
+/**
+ * Attribute module for mfaStatus attribute.
+ * Contains string based on the value of mfaEnforceSettings (ENFORCED_ALL, ENFORCED_PARTIALLY, <empty string>).
+ *
+ * @author Matej Hakoš <492968@mail.muni.cz>
+ */
+public class urn_perun_user_attribute_def_virt_mfaStatus extends UserVirtualAttributesModuleAbstract {
+	private static Logger log = LoggerFactory.getLogger(urn_perun_user_attribute_def_virt_mfaStatus.class);
+
+	@Override
+	public Attribute getAttributeValue(PerunSessionImpl perunSession, User user, AttributeDefinition attribute) {
+		Attribute attr = new Attribute(attribute);
+		String namespace = attribute.getFriendlyName().split(":")[1];
+		String mfaEnforceSettings = AttributesManager.NS_USER_ATTR_DEF + ":mfaEnforceSettings:" + namespace;
+		try {
+			Attribute mfaSettings = perunSession.getPerunBl().getAttributesManagerBl().getAttribute(perunSession, user, mfaEnforceSettings);
+			if (mfaSettings.getValue() == null) {
```

Comment:
```suggestion
			if (mfaSettings.getValue() == null || mfaSettings.getValue() == "") {
```
Pretty sure the value can also be an empty string, no?